api/src/serge/utils/security.py

```diff
@@ -15,15 +16,12 @@
     headers={"WWW-Authenticate": "Bearer"},
 )
 
-pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
-
-
 def verify_password(plain_password, hashed_password):
-    return pwd_context.verify(plain_password, hashed_password)
+    return bcrypt.checkpw(plain_password.encode('utf-8'), hashed_password.encode('utf-8'))
 
 
 def get_password_hash(password):
-    return pwd_context.hash(password)
+    return bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
```

Comment:
This will change the salt every time the function is call.

Reply:
This is the intended behavior. If you're curious, [this is my favorite explanation](https://www.usenix.org/publications/loginonline/bcrypt-25-retrospective-password-security) of salt-in-password hashing.

Reply:
Awesome